### PR TITLE
Added goldpinger helm chart

### PR DESCRIPTION
**Pull request by okgolove**
Signed-off-by: Mikhail Naletov <okgolove@markeloff.net>

#### What this PR does / why we need it:

- add Goldpinger (a tool for network connectivity monitoring between nodes) chart

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/goldpinger/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: goldpinger
+version: 1.0.0
```
Comment (twexler) [inline stable/goldpinger/Chart.yaml:2]: I'd hesitate to set this to 1.0.0 right away.

**Review by twexler — COMMENTED** [review]
Hello! Goldpinger/Helm-charts contributor here! I've got one small comment on this, but everything else looks great! Thanks for building this.

File: stable/goldpinger/Chart.yaml (hunk 1 above)
Comment (okgolove) [reply stable/goldpinger/Chart.yaml:2]: Thank you for the review. Which version should I set?

Usually, reviewers ask me to set version 1.0.0.

Comment (paulczar) [reply stable/goldpinger/Chart.yaml:2]: if we're calling it stable ... then 1.0.0 is good!

**Pull request merged**